File: launchdarkly/ruby/src/GetApplicationVersionsExample.rb
require "json"
require "launchdarkly_client"

LaunchDarklyClient.configure do |config|
    config.api_key["Authorization"] = "YOUR_API_KEY"
end

begin
    response = LaunchDarklyClient::ApplicationsBetaApi.new.get_application_versions(
        "applicationKey_string", # application_key
    )

    p response
rescue LaunchDarklyClient::ApiError => e
    puts "Exception when calling ApplicationsBetaApi#get_application_versions: #{e}"
end

File: launchdarkly/typescript-node/src/GetApplicationVersionsExample.ts
import * as fs from 'fs';
import api from "launchdarkly_client"
import models from "launchdarkly_client"

const apiCaller = new api.ApplicationsBetaApi();
apiCaller.setApiKey(api.ApplicationsBetaApiApiKeys.ApiKey, "YOUR_API_KEY");

apiCaller.getApplicationVersions(
  "applicationKey_string", // applicationKey
  undefined, // filter
  undefined, // limit
  undefined, // offset
  undefined, // sort
).then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling ApplicationsBetaApi#getApplicationVersions:");
  console.log(error.body);
});
